wolfcrypt/src/wc_port.c

```diff
@@ -257,6 +261,14 @@ int wolfCrypt_Init(void)
         }
     #endif
 
+    #ifdef MAXQ10XX_MODULE_INIT
+        ret = maxq10xx_port_init();
+        if (ret != 0) {
+            WOLFSSL_MSG("MAXQ10xx port init failed");
```

Comment:
Prefer WOLFSSL_ERROR_MSG. Trying to encourage its use for error messages.

Reply:
This would look incosistent if we sometimes use WOLFSSL_MSG and other times use WOLFSSL_ERROR_MSG.  

Ah, but I can do it in `maxq10xx.c` and be consistent through the whole file. I will do that.

Reply:
Done in f23cd2a3e .